benchmarks/comparison.py

```diff
@@ -9,14 +9,14 @@
 from django.conf import settings
 
 compare_script = os.getenv('COMPARE_SCRIPT', None)
-compare_command = [compare_script, testjob_before.data.file.name, testjob_after.data.file.name]
+compare_command = [compare_script]
 if not compare_script:
```

Comment:
I have to admit that it is not clear. The comparison is done with compare_script, not compare_command. I can refactor like this if it makes more sense:
`compare_script = os.getenv('COMPARE_SCRIPT', None)
if not compare_script:
    compare_script = os.path.join(settings.EXTERNAL_DIR['BASE'], 'art-testing', 'compare.py')
    compare_command = [compare_script, '--output-for-linaro-automation']
else:
    compare_command = [compare_script]
`